sql/ts/tests/apitests.ts

```diff
@@ -213,26 +213,23 @@ async function testMirroring(skdb: SKDB) {
   const test_pk = {
     table: "test_pk",
     expectedColumns: "(x INTEGER PRIMARY KEY, y INTEGER, skdb_access TEXT)",
+    filterExpr: "x < @thresh",
+    filterParams: { thresh: 43 },
   };
   const view_pk = {
     table: "view_pk",
     expectedColumns: "(x INTEGER, y INTEGER, skdb_access TEXT)",
   };
   await skdb.mirror(test_pk, view_pk);
 
-  const testPkRows = await waitSynch(
-    skdb,
-    "SELECT x,y FROM test_pk",
-    (tail) => tail[0] && tail[0].x == 42,
-  );
+  const testPkRows = await skdb.exec("SELECT x,y FROM test_pk");
   expect(testPkRows).toEqual([{ x: 42, y: 21 }]);
 
-  const viewPkRows = await waitSynch(
-    skdb,
-    "SELECT x,y FROM view_pk",
-    (tail) => tail[0] && tail[0].x == 42,
-  );
-  expect(viewPkRows).toEqual([{ x: 42, y: 63 }]);
+  const viewPkRows = await skdb.exec("SELECT x,y FROM view_pk");
+  expect(viewPkRows).toEqual([
+    { x: 42, y: 63 },
+    { x: 43, y: 66 },
+  ]);
```

Comment:
Not saying I see a problem, but I don't understand if/how these test changes are related.

Reply:
FWIW I tried to document in the commit message: https://github.com/SkipLabs/skdb/pull/67/commits/426a37fc5b6699fe4d2c39a2970eaeabe1e106ef

The idea is that these calls should be local (we want that) and shouldn't need any resiliency around waiting for a result to come in. If they get flaky then something is very wrong.

Reply:
Ah, my bad. I failed to look at the individual commits as the whole thing wasn't overwhelming. :-)